src/main/java/wooteco/subway/service/DefaultFareCalculator.java

```diff
@@ -0,0 +1,32 @@
+package wooteco.subway.service;
+
+import org.springframework.stereotype.Component;
+import wooteco.subway.domain.FareCalculator;
+
+@Component
+public class DefaultFareCalculator implements FareCalculator {
+
+    private static final int DEFAULT_FARE = 1250;
+    private static final int TEN_KILOMETER = 10;
+    private static final int FIFTY_KILOMETER = 50;
+    private static final int DISTANCE_BY_POLICY_FOR_TEN_KILOMETER = 5;
+    private static final int DISTANCE_BY_POLICY_FOR_FIFTY_KILOMETER = 8;
+    private static final int ADDITIONAL_FARE = 100;
+
+    @Override
+    public int calculateFare(int distance, int extraFare) {
+        int fare = DEFAULT_FARE + extraFare;
+        if (TEN_KILOMETER < distance && distance <= FIFTY_KILOMETER) {
+            fare += calculateOverFare(distance - TEN_KILOMETER, DISTANCE_BY_POLICY_FOR_TEN_KILOMETER);
+        }
+        if (distance > FIFTY_KILOMETER) {
+            fare += calculateOverFare(FIFTY_KILOMETER - TEN_KILOMETER, DISTANCE_BY_POLICY_FOR_TEN_KILOMETER);
+            fare += calculateOverFare(distance - FIFTY_KILOMETER, DISTANCE_BY_POLICY_FOR_FIFTY_KILOMETER);
+        }
```

Comment:
다음 정책이 추가되면서 점점 라인이 늘어날 것 같아서 아래와 같은 느낌을 생각했었어요ㅎ
```suggestion
        if (distance > TEN_KILOMETER) {
            fare += calculateOverFare(Math.min(distance - TEN_KILOMETER, FIFTY_KILOMETER - TEN_KILOMETER), DISTANCE_BY_POLICY_FOR_TEN_KILOMETER);
        }
        if (distance > FIFTY_KILOMETER) {
            fare += calculateOverFare(distance - FIFTY_KILOMETER, DISTANCE_BY_POLICY_FOR_FIFTY_KILOMETER);
        }
```